src/publish.proj

```diff
@@ -18,13 +18,14 @@
 
   <PropertyGroup>
     <ContainerName>$(ContainerName.Replace(".","-"))</ContainerName>
+    <PublishBlobFeedUrl>$(PB_PublishBlobFeedUrl)<PublishBlobFeedUrl>
```

Comment:
Please don't directly reference PB_ variables in the build files. They should only be referenced and passed in the build definitions.

Reply:
Just pass in PublishBlobFeedUrl